### PR TITLE
Update SDK to comply with OPENSRC-116

### PR DESCRIPTION
**Pull request by lastperson**
Should be merged after chronicled/open-registry-ethereum#7 is merged and new contracts deployed.

### CODE REVIEW

File: lib/consumer.js [hunk 1]
```diff
@@ -55,7 +53,7 @@ Consumer.prototype.getRegistrar = function() {
 Consumer.prototype.getRegistrant = function(address) {
   var self = this;
   return self.registrar.registrantIndexAsync(address, {from: self.address}).then(function(index) {
```
Comment (drewstone) [inline lib/consumer.js]: These async functions, are they defined in web3 from contract methods now or does Promisify provide them out of the box?

Comment (lastperson) [inline lib/consumer.js]: Promisify provide them when applied to a class or object: https://github.com/chronicled/open-registry-sdk/blob/master/lib/provider.js#L226

File: test/basic-sdk.js [hunk 2]
```diff
@@ -119,6 +119,8 @@ var registry = {
 
   standardSchema: function() {
     invokeCallback(arguments, [null, ['', '', '']]);
+		// Timeout to postpone execution for correct simulation
+    setTimeout(invokeCallback.bind(null, arguments, [null, '0a054261736963123f536368656d612077697468206f6e65206f72206d6f7265206964656e74697469657320616e64206f6e65206e616d6520616e64206465736372697074696f6e1a4c6d657373616765205468696e67207b206f7074696f6e616c20737472696e67206e616d65203d20313b206f7074696f6e616c20737472696e67206465736372697074696f6e203d20323b207d']));
```
Comment (lastperson) [inline test/basic-sdk.js]: It is still invoked right away, on line 121. Also callback parameters in timed out callback doesn't seem right. Probably it is a wrong merge.

Comment (dyro) [inline test/basic-sdk.js]: line#121 won't be executed until the call stack is empty

**Comment by dyro** [comment]
@drewstone or @imolfar: Can you guys take a look at this? It's been open for a while.

**Pull request closed without merging**